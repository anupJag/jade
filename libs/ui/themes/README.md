# fe-themes

This library was generated with [Nx](https://nx.dev).

## Running unit tests

Run `ng test fe-themes` to execute the unit tests via [Jest](https://jestjs.io).
