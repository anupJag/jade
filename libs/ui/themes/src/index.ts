export * from './lib/defaultTheme';
export * from './lib/styled';
export * from './lib/themeProvider';
export { default as styled } from './lib/styled';
export * from './lib/GlobalStyles';
