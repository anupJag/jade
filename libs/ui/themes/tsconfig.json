{
  "extends": "../../../tsconfig.json",
  "compilerOptions": {
    "jsx": "react",
    "allowJs": false,
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "types": ["node", "jest"]
  },
  "include": ["**/*.js", "**/*.jsx", "**/*.ts", "**/*.tsx"]
}
