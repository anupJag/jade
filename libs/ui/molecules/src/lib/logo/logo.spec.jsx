import React from 'react';
import { render } from '@testing-library/react';

import { ThemeProvider } from '@jade/ui/themes';
// TODO : Due to SVG resolving issue in Jest, Its commented
// TODO : It should be enabled once svg support added for JEST
//import { Logo } from './logo';

describe('<Logo />', () => {
  let LogoComponent;
  beforeEach(() => {
    LogoComponent = render(
      <ThemeProvider>
        {/* TODO :  It should be enabled once svg support added for JEST */}
        {/* <Logo /> */}
      </ThemeProvider>,
    );
  });

  /**
   * Capturing Sanpshot : Logo
   *
   *  To check style / structure changes in Styled Components
   */
  test('snapshot check', () => {
    const { asFragment } = LogoComponent;
    expect(asFragment()).toMatchSnapshot();
  });
});
