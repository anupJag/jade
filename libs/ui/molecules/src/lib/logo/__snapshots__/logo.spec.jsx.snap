// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> snapshot check 1`] = `<DocumentFragment />`;
